lib/common/validation/phoneNumber.ts

```diff
@@ -0,0 +1,34 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { IValidator, ValidationResult, Validity } from '@lib/common/validation';
```

Comment:
Put those in a `types.ts` and import the file directly, not the module. This creates a circular import, I'm surprised the linter doesn't complain.